### PR TITLE
Feature: File transfer in secure conversations

### PR DESCRIPTION
**Pull request by vdveer**
* Add files to secure conversations by selecting in app
* Share files (up to mms-sizelimit) and attach them to a secure chat

(removed translations and squashed commits)

### CODE REVIEW

File: src/org/smssecure/smssecure/database/DraftDatabase.java [hunk 1]
```diff
@@ -128,6 +129,7 @@ public String getSnippet(Context context) {
       case IMAGE: return context.getString(R.string.DraftDatabase_Draft_image_snippet);
       case VIDEO: return context.getString(R.string.DraftDatabase_Draft_video_snippet);
       case AUDIO: return context.getString(R.string.DraftDatabase_Draft_audio_snippet);
+      case FILE: return "(file)";
```
Comment (BastienLQ) [inline src/org/smssecure/smssecure/database/DraftDatabase.java]: Missing translation

**Comment by BastienLQ** [comment]
Two things:
* I can't find how to attach a file from SMSSecure: tapping on the paper clip doesn't show a "File" entry in the popup.
* Using OI File Manager, I got `W/AttachmentDraft(28809): Could not match URI to file, ignoring attachment`. It works if I use CM File manager. Bug in your code or in OI File Manager?

**Comment by vdveer** [comment]
There a setting under advanced to activate the file attaching, also it needs to be a secure conversation but from the context I guess you knew that. Maybe the setting can be deleted as I had it in mind to allow multipart sending of bigger files but I didn't have time to complete this yet. 

I will check the OI filemanager as I used a different one to test. Will add a translation for the draft string. Probably overlooked because Android studio showed the string itself instead of the reference to the resource.

**Comment by vdveer** [comment]
My filemanager sent out different Uri's for files which I both need to implement. So bug in my code, not theirs. Will fix it and commit changes to this branch. Also I will remove the setting for disabling files.

**Comment by BastienLQ** [comment]
@vdveer Any news?

**Comment by vdveer** [comment]
Sorry for the absence! Today I have tried to merge back upstream/master in my fork, but I can see there are quite some functional changes in SMSsecure. It will take some time to fix this and look into the way OI filemanager way of sending out URI's of files.

**Pull request closed without merging**